learn/variables/tuple/convert.md

```diff
@@ -0,0 +1,145 @@
+Sometimes it is necessary to convert a tuple to a different data type, such as a list or a string, to perform certain operations. In some cases we need to convert some data types to tuple. Python provides built-in functions for tuple conversion, which can be used to manipulate and process data stored in tuples. Let's explore the various ways of converion and most common cases.
+
+## Tuple vs List in Python
+
+In Python, both tuples and lists are used to store collections of items, but there are some key differences between them. Here are some of the main differences:
+
+- Mutability: Lists are mutable, which means that their contents can be changed, added, or removed after they are created. Tuples, on the other hand, are immutable, which means that their contents cannot be changed once they are created.
```

Comment:
Wrap into `**Bold**` the terms like "Mutability", "Syntax", etc.

Reply:
Done